.github/workflows/deploy-api.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy API to Azure
+
+on:
```

Comment:
Do we still need this action? I'm rewriting the Dockerfile so that we could use only 1 container to deploy both backend and frontend.

Reply:
one PR for one thing, for E2E test, I think we do not need to update `deploy-api.yml`, all right?

Reply:
deleted. It's brought from branch master.